src/script/common/c_content.cpp

```diff
@@ -564,6 +600,8 @@ void push_object_properties(lua_State *L, const ObjectProperties *prop)
 	lua_pushboolean(L, prop->show_on_minimap);
 	lua_setfield(L, -2, "show_on_minimap");
 
+	push_engine_mask(L, prop->engine_mask);
+	lua_setfield(L, -2, "engine_mask");
 	// Remember to update object_property_keys above
```

Comment:
inconsistent spacing (you effectively removed the empty line before the comment)